src/index.html

```diff
@@ -4,9 +4,443 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="logo">
+        <a class="img" href="#home">
+          <img src="images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__link">
+            <a
+              class="nav__adress is-active"
+              href="#apple"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__link">
+            <a class="nav__adress" href="#samsung">Samsung</a>
+          </li>
+
+          <li class="nav__link">
+            <a class="nav__adress" href="#smartphones">Smartphones</a>
+          </li>
+
+          <li class="nav__link">
+            <a
+              class="nav__adress laptop"
+              data-qa="nav-hover"
+              href="#laptopsComputers"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__link">
+            <a class="nav__adress" href="#gadgets">Gadgets</a>
+          </li>
+
+          <li class="nav__link">
+            <a class="nav__adress" href="#tablets">Tablets</a>
+          </li>
+
+          <li class="nav__link">
+            <a class="nav__adress" href="#photo">Photo</a>
+          </li>
+
+          <li class="nav__link">
+            <a class="nav__adress video" href="#video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <article
+      class="main__card card"
+      data-qa="card"
+      >
+        <img
+          class="card__image-block"
+          src="../src/images/imac.jpeg"
+          alt="imac"
+        >
+        <h2 class="card__description-title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__description-code">
+          Product code: 195434
+        </p>
+        <div class="card__review">
+          <div class="card__stars stars--4 stars" >
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+          </div>
```

Comment:
fix using empty lines according to 2nd topic in the [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)